src/components/StatueMenuItem.js

```diff
@@ -1,5 +1,20 @@
 import React from "react";
+import { Link } from "react-router-dom";
+import style from "./StatueMenuItem.module.css";
 
 export function StatueMenuItem(props) {
-  return <li>{props.value.name}</li>;
+  const name = props.value.name;
+  const nameNoSpaces = name.replace(/\s/g, "-");
+  const statueUrl = "/StatueDetailContainer/" + nameNoSpaces;
+
+  return (
+    <li className={style.li}>
+      <Link
+        to={{ pathname: statueUrl, state: props.value }}
```

Comment:
is the state required anymore now that the routes render their own child `statueDetailContainers`?